pkg/config/validator.go

```diff
@@ -264,14 +264,14 @@ func (a adapterKey) String() string {
 // compatfilterConfig
 // given a yaml file, filter specific keys from it
 // globalConfig contains descriptors and adapters which will be split shortly.
-func compatfilterConfig(cfg string, shouldSelect func(string) bool) ([]byte, map[string]interface{}, error) {
+func compatfilterConfig(cfg string, shouldSelect func(string) bool) ([]byte, error) {
```

Comment:
Something required you to add a param?

Reply:
unparam complained the parameter is not used.